### PR TITLE
Hybridize

### PR DESCRIPTION
**Pull request by danyakarras**

### CODE REVIEW

**Comment by danyakarras** [comment]
@dbatiste here is the PR

File: d2l-progress.html [hunk 1]
```diff
@@ -8,8 +9,11 @@
 			-moz-appearance: none;
 			-webkit-appearance: none;
 			background-color: var(--d2l-color-woolonardo);
-			border: 1px solid var(--d2l-color-gypsum);
 			border-radius: 10px;
+			border: none;
+			box-shadow: inset 0px 2px 0px 0px var(--d2l-color-gypsum);
+			height:1rem;
```
Comment (dbatiste) [inline d2l-progress.html]: Super nitpick: although there is LOTs of older CSS that doesn't follow this convention, we try to format CSS with a space following the colon.  :)

**Comment by dbatiste** [comment]
Looks good Danya.  Were you able to make any progress on the Galen front?  If not, I can probably get something going for that - I think the boiler plate stuff is there.

It would be good to get DL's eyes on this too since he's a lot more familiar with the hybrid approach.  @dlockhart : fyi.

File: bower.json [hunk 2]
```diff
@@ -16,10 +16,30 @@
...
+        "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0"
+      },
+	  "resolutions": {
+		"webcomponentsjs": "^v0.7.0"
+	  }
+    }
+  },
+  "resolutions": {
+    "webcomponentsjs": "^v1.0.0",
+	"polymer": "^2.0.0"
```
Comment (dlockhart) [inline bower.json]: Is the `polymer ^2.0.0` resolution needed? I haven't need that before or seen it elsewhere.

Comment (danyakarras) [inline bower.json]: I don't think it is. I'll take it out and try again!

File: d2l-progress.html (hunk 1 above)
Comment (danyakarras) [inline d2l-progress.html]: yes! I must have missed that one. thanks

**Comment by danyakarras** [comment]
I wasn't able to get Galen set up (still not sure what exactly it is), but feel free to either set up, or I can start Googling :)

**Comment by dbatiste** [comment]
Ok.  Most of the Galen stuff is set-up... just needs to be adjusted a bit for progress since I copied from another repo.  I'll get it going tonight if I get some time.

**Comment by danyakarras** [comment]
Thanks, Dave B! Let me know if there's anything I can do.

**Pull request merged**